src/Host/Controllers/Identity/RolesController.cs

```diff
@@ -29,11 +29,11 @@ public Task<RoleDto> GetByIdWithPermissionsAsync(string id, CancellationToken ca
         return _roleService.GetByIdWithPermissionsAsync(id, cancellationToken);
     }
 
-    [HttpPut("{id}/permissions")]
+    [HttpPut("permissions")]
```

Comment:
we'll have to leave the id here... otherwise there's no way to distinguish between update role and update rolepermissions... do it like in brands and products controller... compare route id with request id and return badrequest when they are not the same